### PR TITLE
Create master branch docker images

### PR DESCRIPTION
**Pull request by mlautman**
### Description

Creating master branch docker images so that we can test against the most recent version of master without building moveit in travis. 

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extended the tutorials / documentation, if necessary [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Include a screenshot if changing a GUI
- [ ] Document API changes relevant to the user in the moveit/MIGRATION.md notes
- [ ] Created tests, which fail without this PR [reference](http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/tests.html)
- [ ] Decide if this should be cherry-picked to other current ROS branches
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: .docker/ci/Dockerfile [hunk 1]
```diff
@@ -17,10 +17,10 @@ RUN \
     cd src && \
     #
     # Download moveit source, so that we can get necessary dependencies
-    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
+    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit/master/moveit.rosinstall && \
     #
     # Update apt package list as previous containers clear the cache
-    apt-get -qq update && \
+    apt-get -qq update --fix-missing && \
```
Comment (rhaschke) [inline .docker/ci/Dockerfile]: Why do you think, it's valuable to ignore missing packages?
Shouldn't we get noticed about this issue?

File: .docker/source/Dockerfile [hunk 2]
```diff
@@ -27,4 +27,5 @@ ENV PYTHONIOENCODING UTF-8
 RUN catkin config --extend /opt/ros/$ROS_DISTRO --install --cmake-args -DCMAKE_BUILD_TYPE=Release
     # Status rate is limited so that just enough info is shown to keep Docker from timing out,
     # but not too much such that the Docker log gets too long (another form of timeout)
-    # catkin build --limit-status-rate 0.001 --no-notify
+
+RUN catkin build --limit-status-rate 0.001 --no-notify
```
Comment (rhaschke) [inline .docker/source/Dockerfile]: I think, originally this command was part of the previous RUN command.
If you like to separate them by now, you should adjust the indentation of the comment above.

Comment (rhaschke) [reply .docker/source/Dockerfile]: Please consider https://github.com/ros-planning/moveit/pull/1207. Although I didn't like it, Dave had a reason to remove the build step.

File: .docker/ci/Dockerfile (hunk 1 above)
Comment (mlautman) [reply .docker/ci/Dockerfile]: I was going by the console output from a failed docker build which recommended --fix-missing. I didn't realize that by fix they actually meant ignore haha. I have reverted this change

File: .docker/source/Dockerfile [hunk 3]
```diff
@@ -27,4 +27,4 @@ ENV PYTHONIOENCODING UTF-8
 RUN catkin config --extend /opt/ros/$ROS_DISTRO --install --cmake-args -DCMAKE_BUILD_TYPE=Release
```
[the remark this answers is not in the document]
Comment (mlautman) [reply .docker/source/Dockerfile]: Thanks!

**Pull request merged**